lib/cobra_commander/component_tree.rb

```diff
@@ -93,6 +99,14 @@ def gemfile_path
         def gemfile_lock_path
           File.join(@root_path, "Gemfile.lock")
         end
+
+        def bundler_version_supporting_path_method?
+          @bundler_version_supporting_path_method ||= bundler_minor_version >= 16
```

Comment:
I would actually do a comparison on the whole version. There’s a helper method somewhere in Rubygems (maybe Gem::Version comparison operator) to do that more accurately than ignoring the major version.